src/components/modules/toolbar/index.ts

```diff
@@ -335,11 +406,55 @@ export default class Toolbar extends Module<ToolbarNodes> {
     $.append(this.Editor.UI.nodes.wrapper, this.nodes.wrapper);
   }
 
+  /**
+   * Creates the Toolbox instance and return it's rendered element
+   */
+  private makeToolbox(): Element {
+    /**
+     * Make the Toolbox
+     */
+    this.toolboxInstance = new Toolbox({
+      api: this.Editor.API.methods,
+      tools: this.Editor.Tools.blockTools,
+      shortcutsScopeElement: this.Editor.UI.nodes.redactor,
```

Comment:
Might be better to provide some nodes from Editor API
```
editorAPI.nodes.redactor

// or

editorAPI.ui.redactor
```

Reply:
done